utils/qr_code_generator.py

```diff
@@ -0,0 +1,19 @@
+import pyqrcode
+from .constants import IMAGES_DIR
```

Comment:
This needs to be `from utils.constants import IMAGES_DIR`. This relies on `PYTHONPATH`.

Reply:
Addressed in  0e285a0